verl/trainer/config/rollout/rollout.yaml

```diff
@@ -103,6 +106,9 @@ engine_kwargs:
 # Sampling parameters used during validation.
 val_kwargs:
 
+  # Required when using verl.utils.omega_conf_to_dataclass to instantiate dataclass configs
+  _target_: verl.workers.config.SamplingConfig
```

Comment:
Should we move rollout sampling params to `SamplingConfig`?

Reply:
Yes. I left a TODO. Currently, we don't want to introduce breaking changes. RolloutConfig should match rollout.yaml exactly

Reply:
We also want to remove the logprob bsz into actor.infer in the following PRs